thinc/util.py

```diff
@@ -279,7 +277,7 @@ def xp2tensorflow(
 def tensorflow2xp(tensorflow_tensor: "tf.Tensor") -> Array:  # pragma: no cover
     """Convert a Tensorflow tensor to numpy or cupy tensor."""
     assert_tensorflow_installed()
-    if "GPU" in tensorflow_tensor.device:
+    if "GPU" in tensorflow_tensor.device and has_tfdlpack:
         return cupy.fromDlpack(tfdlpack.to_dlpack(tensorflow_tensor))
```

Comment:
I think we should abandon DLPack and just find a different way to copy the tensor over to cupy.

Reply:
Okay, I'm not sure what a replacement would look like, do you want me to remove it for now and someone else can add a replacement at a later point? Or do you want to prescribe a set of changes and a replacement library for this PR?

Reply:
Let's just remove it. Since it's a small change and the time difference makes this a bit async, I'll just merge and adjust :)